### PR TITLE
fix: remove usage of getExchangeFeeRate via FeePool

### PR DESCRIPTION
**Pull request by evgenyboxer**

### CODE REVIEW

File: src/pages/Trade/components/ChartCard/ChartCard.tsx [hunk 1]
```diff
@@ -92,6 +92,7 @@ const ChartCard: FC<ChartCardProps> = memo(({ synthPair }) => {
 					<Periods>
 						{PERIOD_LABELS.map((period) => (
 							<Button
+								key={period.value}
```
Comment (evgenyboxer) [inline src/pages/Trade/components/ChartCard/ChartCard.tsx:95]: unrelated fix... saw this as a warning in the console

File: src/pages/Trade/components/CreateOrderCard/CreateOrderCard.js [hunk 2]
```diff
@@ -67,7 +65,7 @@ const CreateOrderCard = ({
 	const { colors } = useContext(ThemeContext);
 	const [baseAmount, setBaseAmount] = useState(INPUT_DEFAULT_VALUE);
 	const [quoteAmount, setQuoteAmount] = useState(INPUT_DEFAULT_VALUE);
-	const [feeRate, setFeeRate] = useState(exchangeFeeRate);
+	const [feeRate, setFeeRate] = useState(0);
```
Comment (evgenyboxer) [inline src/pages/Trade/components/CreateOrderCard/CreateOrderCard.js:68]: as discussed @clementbalestrat I saw this to 0 by default.

**Comment by clementbalestrat** [comment]
Let's merge it once we've got a version deployed on testnet
